crates/mdk-core/src/key_packages.rs

```diff
@@ -2029,4 +2109,333 @@ mod tests {
+            hex_mdk
+                .parse_serialized_key_package(&hex_key_package, ContentEncoding::Hex)
+                .is_ok(),
+            "Hex MDK should parse hex key package with Hex encoding"
+        );
+        assert!(
+            hex_mdk
+                .parse_serialized_key_package(&base64_key_package, ContentEncoding::Base64)
+                .is_ok(),
+            "Hex MDK should parse base64 key package with Base64 encoding"
+        );
+        assert!(
+            base64_mdk
+                .parse_serialized_key_package(&hex_key_package, ContentEncoding::Hex)
+                .is_ok(),
+            "Base64 MDK should parse hex key package with Hex encoding"
+        );
+        assert!(
+            base64_mdk
+                .parse_serialized_key_package(&base64_key_package, ContentEncoding::Base64)
+                .is_ok(),
+            "Base64 MDK should parse base64 key package with Base64 encoding"
+        );
+    }
+
+    #[test]
+    fn test_key_package_size_comparison() {
+        let test_pubkey =
+            PublicKey::from_hex("884704bd421671e01c13f854d2ce23ce2a5bfe9562f4f297ad2bc921ba30c3a6")
+                .unwrap();
+        let relays = vec![RelayUrl::parse("wss://relay.example.com").unwrap()];
+
+        // Create with hex
+        let hex_config = crate::MdkConfig {
+            use_base64_encoding: false,
+        };
+        let hex_mdk = crate::tests::create_test_mdk_with_config(hex_config);
+
+        let (hex_key_package, _) = hex_mdk
+            .create_key_package_for_event(&test_pubkey, relays.clone())
+            .expect("Failed to create hex key package");
+
+        // Create with base64
+        let base64_config = crate::MdkConfig {
+            use_base64_encoding: true,
+        };
+        let base64_mdk = crate::tests::create_test_mdk_with_config(base64_config);
+
+        let (base64_key_package, _) = base64_mdk
+            .create_key_package_for_event(&test_pubkey, relays)
+            .expect("Failed to create base64 key package");
+
+        let hex_size = hex_key_package.len();
+        let base64_size = base64_key_package.len();
+
+        // Base64 should be smaller than hex
+        assert!(
+            base64_size < hex_size,
+            "Base64 ({} bytes) should be smaller than hex ({} bytes)",
+            base64_size,
+            hex_size
+        );
+
+        // Calculate the savings
+        let savings_percent = ((hex_size - base64_size) as f64 / hex_size as f64) * 100.0;
+        println!(
+            "Size comparison: hex={} bytes, base64={} bytes, savings={:.1}%",
+            hex_size, base64_size, savings_percent
+        );
+
+        // Base64 should be approximately 33% smaller (hex is 2x, base64 is 1.33x)
+        // Allow some variance due to encoding overhead
+        assert!(
+            savings_percent > 25.0 && savings_percent < 40.0,
+            "Expected savings between 25-40%, got {:.1}%",
+            savings_percent
+        );
+    }
```

Comment:
I don't think this test is really useful. This is basically testing a property of the encodings themselves and not our code. I'd just remove it entirely